package/boot/uboot-mediatek/Makefile

```diff
@@ -461,6 +461,18 @@ define U-Boot/mt7981_qihoo_360t7
   DEPENDS:=+trusted-firmware-a-mt7981-spim-nand-ddr3
 endef
 
+define U-Boot/mt7981_snr_cpe-ax2
+  NAME:=snr CPE-AX2
```

Comment:
SNR should be written in capital:
![image](https://github.com/user-attachments/assets/1c6c2d42-4cdc-4b5e-847a-fa493b3561c2)

And probably SNR is a vendor. SNR-CPE-AX2 is a model.
In this case here and also in all other places:
`U-Boot/mt7981_snr_snr-cpe-ax2`
`NAME:=SNR SNR-CPE-AX2`